packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.ts

```diff
@@ -24,13 +24,13 @@ export const appBadges = `
   </mj-section>
   <mj-section>
     <mj-column>
-      <% if (onDesktopOrTabletDevice) { %>
+      <% if (typeof(onDesktopOrTabletDevice) == "undefined" || onDesktopOrTabletDevice) { %>
```

Comment:
This appears to be both checking that `onDesktopOrTabletDevice` is either undefined or not. What's it meant to be doing?

Reply:
We should also always use the triple equals when doing a comparison like this